doc/tutorial/tutorial.mdk

```diff
@@ -3270,9 +3259,9 @@ following form---they transform pure post-conditions to pure
 pre-conditions.
 
 ```
-kind PurePost (a:Type) = a -> Type
-kind PurePre = Type
-kind PureWP (a:Type) = PurePost a -> PurePre
+let pure_post (a:Type) = a -> Type
+let pure_pre = Type
+let pure_wp (a:Type) = pure_post a -> pure_pre

```

Comment:
Would this be more readable with `type pure_post …` instead of `let pure_post …`?